### PR TITLE
Fixes #19534: All .cf and .st files under in rudder-techniques should have an updated license header

### PR DESCRIPTION
**Pull request by Fdall**
https://issues.rudder.io/issues/19534

### CODE REVIEW

File: qa-test [hunk 1]
```diff
@@ -28,9 +28,23 @@ test_shell_error()
     done
 }
 
+# verify that each file has a correct license header
+test_license_header()
+{
+  find . -name '*.cf' -print0 |
+    while IFS= read -r -d '' file; do
+      if ! grep -qE "# Copyright 20[0-9]{2} Normation SAS" $file; then
```
Comment (amousset) [inline qa-test:36]: 2100 limit seems fair

**Comment by Fdall** [comment]
PR updated with a new commit

**Comment by Fdall** [comment]
PR updated with a new commit

**Comment by Fdall** [comment]
PR updated with a new commit

**Comment by Fdall** [comment]
PR rebased

**Comment by Fdall** [comment]
PR updated with a new commit

**Comment by Fdall** [comment]
PR updated with a new commit

**Comment by Fdall** [comment]
PR updated with a new commit

**Comment by Fdall** [comment]
OK, squash merging this PR

**Comment by Fdall** [comment]
OK, merging this PR

**Pull request merged**